daemon/Cargo.toml

```diff
@@ -20,20 +20,7 @@ notify = "4.0.15"
 distill-downstream-lmdb-rkv = "0.11.0-windows-fix"
 rayon = { version = "1.3", optional = true }
 log = { version = "0.4", features = ["serde", "std"] }
-tokio = { version = "1.2", features = [
-    "net",
-    "fs",
-    "sync",
-    "time",
-    "rt",
-    "rt-multi-thread",
-    "io-util",
-    "macros",
-] }
-tokio-util = { version = "0.6.1", features = ["compat"] }
-futures = "0.3"
-async-lock = { version = "2.1" }
-async-channel = { version = "1.4" }
+futures = { version = "0.3", default-features = false, features = ["std", "async-await"] }
```

Comment:
I'm not sure what's better -- removing `async-lock` and `async-channel` because we can just use the features in `futures` _OR_ paring down `futures` to the absolute minimum (no `std`) and then then using `async-lock` and `async-channel` instead of `futures`. Any thoughts?
@kabergstrom @aclysma

Reply:
I think technically this is already reduced surface area because the previous cargo was just a blanket `futures = "0.3"`.